services/ci-release/README.md

```diff
@@ -0,0 +1,10 @@
+# ci-release
+
+This folder holds the assets for `ci-release` project.
```

Comment:
This is not a very useful description. What's the purpose of the `ci-release` project? Even the name is not very informative.

Reply:
I know it is not that informative because I do not know anything about it.
I need this because the BW script is broken (i think to everyone, including Clayton after https://jira.coreos.com/browse/DPTP-645?focusedCommentId=142458&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-142458) without impersonating.

Let us ask @smarterclayton since he added the secrets to the BW script.

`oc get all -n  ci-release` shows lots of things that look very important, but i do not know who would miss it. :smile:

Reply:
oh, it's not a new namespace, sorry 🤦‍♂